PersonApi/Startup.cs

```diff
@@ -71,7 +72,9 @@ public void ConfigureServices(IServiceCollection services)
                 })
                 .SetCompatibilityVersion(CompatibilityVersion.Version_3_0);
 
+            services.AddFluentValidation();
```

Comment:
This line is only required if validators are implemented _in this assembly_ (which I don't think is the case).